src/Preprocessor.zig

```diff
@@ -1785,11 +1865,15 @@ fn pragma(pp: *Preprocessor, tokenizer: *Tokenizer, pragma_tok: RawToken, operat
     }, pragma_name_tok.expansionSlice());
 }
 
-fn findIncludeSource(pp: *Preprocessor, tokenizer: *Tokenizer) !Source {
+fn findIncludeFilenameToken(
+    pp: *Preprocessor,
+    first: *RawToken,
```

Comment:
Why is `first` passed as a pointer?

Reply:
That's probably a holdover from how the code was original constructed; it looks like I can just pass the RawToken in and make a copy of it in the function since the `macro_string` mutation doesn't seem like it needs to escape this function.